tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Resource    ../../../Resources/ODS.robot
+Resource    ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Resource    ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+
+
+*** Test Cases ***
+Any OpenShift User Should Also Be A RHODS User
+    [Documentation]    Any OpenShift user should also be a RHODS user
+    [Tags]    Sanity
+    ...       ODS-1164
+    Set Library Search Order    SeleniumLibrary    InstalledOperators
+    Open Browser    ${OCP_CONSOLE_URL}    ${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To Openshift    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}    ${OCP_ADMIN_USER.AUTH_TYPE}
+    Navigate To Page    Workloads    ConfigMaps
+    Select Rhods-Groups-Config
+    Verify Groups ConfigMap Contains The Expected OCP Groups
+    Spawn A Minimal Notebook
```

Comment:
Do you need to actually spawn a notebook or do you just want to check that user can log in to jupyterhub? if so, you can take a look at `special-user-testing.robot` in `tests/Tests/500__jupyterhub`

Reply:
I should check if we can span a notebook.